x/inflation/keeper/migrations.go

```diff
@@ -1,5 +1,14 @@
 package keeper
 
+import (
+	sdk "github.com/cosmos/cosmos-sdk/types"
+	"github.com/cosmos/cosmos-sdk/types/module"
+
+	v2 "github.com/tharsis/evmos/v2/x/inflation/migrations/v2"
```

Comment:
@fedekunze This import is isn't found locally yet and breaks the tests: 
`cannot find module providing package github.com/tharsis/evmos/v2/x/inflation/migrations/v2`